repotracker/repotracker.go

```diff
@@ -920,20 +920,38 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 	}))
 	batchTimeCatcher := grip.NewBasicCatcher()
 	debuggingData := map[string]string{}
+	var githubCheckAliases model.ProjectAliases
+	if v.Requester == evergreen.RepotrackerVersionRequester && projectInfo.Ref.GithubChecksEnabled {
+		githubCheckAliases, err = model.FindAliasInProject(v.Identifier, evergreen.GithubChecksAlias)
+		grip.Error(message.WrapError(err, message.Fields{
+			"message": "error getting github check aliases",
+			"project": projectInfo.Project.Identifier,
+			"version": v.Id,
+		}))
+	}
 	for _, buildvariant := range projectInfo.Project.BuildVariants {
-		taskNames := pairsToCreate.TaskNames(buildvariant.Name)
+		taskNames := pairsToCreate.TaskNames(buildvariant.Name) // if no aliases provided then this is empty
+		var aliasesMatchingVariant model.ProjectAliases
+		aliasesMatchingVariant, err = githubCheckAliases.AliasesMatchingVariant(buildvariant.Name, buildvariant.Tags)
```

Comment:
How do you feel about having all tasks be check tasks if there is no github check alias (or maybe if it exists but is empty)?

Reply:
I don't totally understand the logic of what you're describing. If there are no aliases matching the variant, then I imagine that variant has been left out of the aliases on purpose.

Reply:
I'm talking about if you have no __github_check alias at all. If I just check the box to see what it does, but not define any alias, I might expect that the next commit considers every task for checks. Right now it considers no tasks

Reply:
Ah, there's validation to ensure that there are github_check aliases, if the box is checked. 
https://github.com/ablack12/evergreen/blob/49c7a2b8b30084c7973337bbb11986e3dac86756/service/project.go#L418
I thought about not requiring github check aliases, but it adds a bit of complexity, isn't consistent with how other aliases work, and probably isn't something people would want (and if they did, the all tasks/variants regex is simple).